x-pack/plugin/spatial/src/test/java/org/elasticsearch/xpack/spatial/index/mapper/GeoShapeWithDocValuesFieldMapperTests.java

```diff
@@ -254,7 +254,7 @@ public void testIgnoreMalformedValues() throws IOException {
             ParsedDocument document = ignoreMapper.parse(sourceToParse);
             assertThat(document.docs().get(0).getFields("field").length, equalTo(0));
             MapperParsingException exception = expectThrows(MapperParsingException.class, () -> failMapper.parse(sourceToParse));
-            assertThat(exception.getCause().getMessage(), containsString("at least 4 polygon points required"));
+            assertThat(exception.getCause().getMessage(), containsString("Unable to Tessellate shape"));
```

Comment:
Since the PR description seems to indicate this change leads to better error messages, I wonder if there is a possible test that could be written to show that case? The test here seems to have a less useful error message for the end-user.

Reply:
Yes, that is the reason I still have it as a draft. Soon we will be able to change it:

https://issues.apache.org/jira/browse/LUCENE-10437